src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesComputer/sidepanel2.jelly

```diff
@@ -27,4 +27,5 @@ THE SOFTWARE.
   <l:task icon="icon-clipboard icon-md" href="${rootURL}/${it.url}log" title="${%Log}" permission="${app.ADMINISTER}"/>
   <l:task icon="icon-computer icon-md" href="${rootURL}/${it.url}podLog" title="${%Pod Log}" permission="${app.ADMINISTER}"/>
   <l:task icon="icon-computer icon-md" href="${rootURL}/${it.url}events" title="${%Events}" permission="${app.ADMINISTER}"/>
```

Comment:
probably

```suggestion
  <l:task icon="icon-computer icon-md" href="${rootURL}/${it.url}podLog" title="${%Pod Log}" permission="${it.EXTENDED_READ_AND_CONNECT}"/>
  <l:task icon="icon-computer icon-md" href="${rootURL}/${it.url}events" title="${%Events}" permission="${it.EXTENDED_READ_AND_CONNECT}"/>
```
as well, though then https://github.com/jenkinsci/kubernetes-plugin/blob/c3f2a06be34e69614cc8de72340f63cc78f8fbe7/src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesComputer/podLog.jelly#L27 and https://github.com/jenkinsci/kubernetes-plugin/blob/c3f2a06be34e69614cc8de72340f63cc78f8fbe7/src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesComputer/events.jelly#L27 would need to be changed to match

Reply:
Done in https://github.com/jenkinsci/kubernetes-plugin/pull/1228/commits/5c620515ce125787015532a2ce0f0640f99a641a

Reply:
Only the Jelly parts, but you would also need to adjust the Java permissions to match, or under some conditions you would be exposing links which a user could click on only to get a 403.

Reply:
Oh, I misunderstood the suggestion. I need help on this one then as I have no idea what to do and where to look.

Reply:
Sorry, I got confused, this is not about checks from Java code and you did do this.